src/main/java/network/radicle/tools/github/providers/JsonMapperCustomizer.java

```diff
@@ -0,0 +1,20 @@
+package network.radicle.tools.github.providers;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.quarkus.jackson.ObjectMapperCustomizer;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class JsonMapperCustomizer implements ObjectMapperCustomizer {
+
+    @Override
+    public void customize(ObjectMapper objectMapper) {
+        objectMapper.findAndRegisterModules()
```

Comment:
`findAndRegisterModules` should already register any modules in jvm mode.
However, when running in native mode, this might need to be removed completely.